spec/services/migration/migrators/user_spec.rb

```diff
@@ -37,21 +37,22 @@ def setup_failure_state
         expect(failure_manager).not_to have_received(:record_failure)
       end
 
-      it "records a failure when there are multiple emails" do
-        ecf_user = create(:ecf_migration_user, :npq, email: "email-1@example.com")
-        participant_identity = create(:ecf_migration_participant_identity, user: ecf_user, email: "email-2@example.com", external_identifier: SecureRandom.uuid)
-        create(:ecf_migration_npq_application, teacher_reference_number: ecf_user.npq_applications.first.teacher_reference_number, participant_identity:)
-
-        instance.call
-        expect(failure_manager).to have_received(:record_failure).with(ecf_user, /There are multiple different emails from user identities in NPQ applications/)
-      end
-
-      it "records a failure when existing email in npq does not match ecf email" do
-        ecf_user = create(:ecf_migration_user, :npq, email: "email-1@example.com")
-        create(:user, ecf_id: ecf_user.id, email: "email-2@example.com")
+      it "sets User email from recent updated npq application participant identity" do
```

Comment:
It looks like we have two very similar tests in here:

```
sets User email from recent updated npq application participant identity
sets User attributes from recent updated npq application
```

Is it worth combining them?

Reply:
Combined tests 👍

Reply:
It looks like both are still there? I think we can ditch `sets User attributes from recent updated npq application` now as your other test covers it? although it might be worth having a separate test that makes sure we set the email to the `user.email` if there's no application -> participant identity -> email

Reply:
I don't think its a real scenario. Participant Identity is required for NPQ Application and email is required for Participant Identity. We won't run the migration on the user if it doesn't have applications.

Reply:
Is it worth ditching the `|| user.email` fallback in that case if its redundant?

Reply:
removed